components/sidebar/invite_members_button.test.tsx

```diff
@@ -49,14 +49,19 @@ describe('components/sidebar/invite_members_button', () => {
         },
     };
 
+    const props = {
+        onClick: jest.fn(),
+        touchedInviteMembersButton: false,
```

Comment:
Could we add a test here for when it is touched? `touchedInviteMembersButton: true`